services/environment-checker.js

```diff
@@ -0,0 +1,19 @@
+function EnvironmentChecker(env, logger, deprecated) {
+  this.deprecated = deprecated;
+
+  this.getActives = () => this.deprecated.filter(variable => env[variable] !== undefined);
+
+  this.logWarnings = () => {
+    const activeEnvironmentVariables = this.getActives();
+    if (activeEnvironmentVariables.length) {
+      logger.warn(
+        'Environment variables usage detected:',
+        activeEnvironmentVariables.map(variable => ` - ${variable}`),
```

Comment:
Is it on purpose that you don't insert new lines?

Reply:
Hum, no 😅. Fixed. Thank you!